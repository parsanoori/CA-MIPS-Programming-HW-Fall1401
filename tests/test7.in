4
354594007
675654636
123721778
492084872