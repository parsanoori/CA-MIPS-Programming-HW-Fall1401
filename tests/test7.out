123721778
354594007
492084872
675654636